ynr/apps/candidates/tests/test_constituency_view.py

```diff
@@ -96,7 +96,7 @@ def test_any_constituency_page_without_login(self):
 
     def test_any_constituency_page(self):
         # Just a smoke test for the moment:
-        with self.assertNumQueries(46):
+        with self.assertNumQueries(48):
```

Comment:
I'm confused about why removing the one-to-one and generic relations here means we're making _more_ queries. Did you look into this at all?